Competition2018/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -23,6 +23,15 @@ public void rise(){
 		
 	}
 	
+
+	/**
+	 * Returns true if the robot is rising
```

Comment:
The _robot_ is rising? I didn't know we were building a flying machine :laughing: I think you might've been referring to the elevator here and below.

Reply:
Great point! I will fix it! :)